File: Lab01/01/arrSum.cpp
#include <iostream>

int sumarArreglo(int arr[], int tamanio){
    int res = 0;
    for(int i = 0; i < tamanio; i++){
        res += arr[i];
    }
    return res;
}

int sumarArregloDinamico(int* arr, int tamanio){
    int res = 0;
    for(int i = 0; i < tamanio; i++){
        res += arr[i];
    }
    return res;
}

int main(){
    int myArr[] = {1, 2, 3, 4, 5};
    int suma = sumarArreglo(myArr, 5);
    std::cout << "La suma es: " << suma << std::endl;

    int tam = 5;
    int* myArrDin = new int[tam];
    for(int i = 0; i < tam ; i++){
        myArrDin[i] += i + 1;
    }
    int suma2 = sumarArregloDinamico(myArrDin, tam);
    std::cout << "La suma del arreglo dinamico es: " << suma2 << std::endl;
}

File: Lab01/01/arrSum.java
public class arrSum{
    public static int sumarArreglo(int[] arr){
        int suma = 0;
        for(int i = 0; i < arr.length; i++){
            suma += arr[i];
        }
        return suma;
    }
    
    public static void main(String[] args){
        int[] myArr = {1, 2, 3, 4, 5};
        int res = sumarArreglo(myArr);
        System.out.println("La suma es: " + res);
    }
}